package.json

```
@@ -5,12 +5,12 @@
   "main": "index.js",
   "scripts": {
     "build": "npm-run-all build:* --sequential --print-name",
-    "build:rspack": "RSPACK=1 pnpm rspack -c ./rspack.config.mjs",
-    "build:rsbuild": "rsbuild build",
-    "build:webpack": "WEBPACK=1 pnpm webpack -c ./rspack.config.mjs",
-    "dev:rspack": "RSPACK=1 rspack dev -c ./rspack.config.mjs",
+    "build:rspack": "cross-env RSPACK=1 pnpm rspack -c ./rspack.config.mjs",
+    "build:rsbuild": "rsbuild build --env-mode development",
```

I prefer to keep the normal `rsbuild build` here

```suggestion
    "build:rsbuild": "rsbuild build",
```